parts/k8s/addons/ip-masq-agent.yaml

```diff
@@ -18,6 +18,13 @@ spec:
       hostNetwork: true
       nodeSelector:
         beta.kubernetes.io/os: linux
+      tolerations:
+      - key: CriticalAddonsOnly
```

Comment:
Just some questions, what is this CriticalAddonsOnly tolerance used for? Is it needed here or just follow some conventions?

The PR looks good to me.